tests/unit/models/LifecycleConfiguration.js

```diff
@@ -173,99 +252,111 @@ function generateXml(errorTag, tagValue) {
     return Lifecycle;
 }
 
-function generateParsedXml(errorTag, tagValue, cb) {
-    const xml = generateXml(errorTag, tagValue);
+function generateParsedXml(errorTag, tagObj, cb) {
+    const xml = generateXml(errorTag, tagObj);
     parseString(xml, (err, parsedXml) => {
         assert.equal(err, null, 'Error parsing xml');
         cb(parsedXml);
     });
 }
 
 function checkError(parsedXml, error, errMessage, cb) {
-    const lifecycleConfig =
-        new LifecycleConfiguration(parsedXml).
+    const lcConfig = new LifecycleConfiguration(parsedXml).
         getLifecycleConfiguration();
-    assert.strictEqual(
-        lifecycleConfig.error[error], true);
-    assert.strictEqual(
-        lifecycleConfig.error.description, errMessage);
+    assert.strictEqual(lcConfig.error[error], true);
+    assert.strictEqual(lcConfig.error.description, errMessage);
     cb();
 }
 
-describe('LifecycleConfiguration class getLifecycleConfiguration', () => {
-    const tagValue = {};
-    requiredTags.forEach(tag => {
-        it(`should return MalformedXML error if ${tag.tag} tag is missing`,
-        done => {
-            generateParsedXml(tag.tag, tagValue, parsedXml => {
-                checkError(parsedXml, 'MalformedXML', tag.errMessage, done);
-            });
-        });
+describe.only('LifecycleConfiguration class getLifecycleConfiguration', () => {
```

Comment:
:thinking: sneaky

Reply:
omg i always do that

Reply:
same, hopefully the linter gets added soon xD